diff --git a/buildsystem/get-vlc.sh b/buildsystem/get-vlc.sh
index 9caceb3..76012b9 100755
--- a/buildsystem/get-vlc.sh
+++ b/buildsystem/get-vlc.sh
@@ -69,7 +69,7 @@ done
 # Fetch VLC source #
 ####################
 
-VLC_TESTED_HASH=f1ed769f555d083c876b5338ed41b64f57e0d48a
+VLC_TESTED_HASH=9c4768291ee0ce8e29fdadf3e05cbde2714bbe0c
 VLC_REPOSITORY=https://code.videolan.org/videolan/vlc.git
 VLC_BRANCH=3.0.x
 if [ ! -d "vlc" ]; then
